### PR TITLE
export GOFLAGS

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
Fix go get: disabled by -mod=vendor
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/8863/rehearse-8863-pull-ci-openshift-cincinnati-operator-release-4.5-cincinnati-operator-e2e-aws/2

### CODE REVIEW

**Comment by pawanpinjarkar** [comment]
Ref https://stackoverflow.com/questions/58496567/how-can-i-fix-go-get-disabled-by-mod-vendor

File: hack/functest.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
 	echo "Openshift CI detected, deploying using image $CATALOG_FULL_IMAGE_NAME"
 fi
 
+export GOFLAGS=""
```
Comment (rwsu) [inline hack/functest.sh:11]: I would put this export within this block just before the first "go get":

````
if ! [ -x "$GINKGO" ]; then
	echo "Retrieving ginkgo and gomega build dependencies"
	go get github.com/onsi/ginkgo/ginkgo
	go get github.com/onsi/gomega/...
else
````

Comment (pawanpinjarkar) [reply hack/functest.sh]: Done

File: hack/functest.sh [hunk 2]
```diff
@@ -12,6 +12,7 @@ GOBIN="${GOBIN:-$GOPATH/bin}"
 GINKGO=$GOBIN/ginkgo
 
 if ! [ -x "$GINKGO" ]; then
+	export GOFLAGS=""
```
Comment (sacharya) [inline hack/functest.sh:15]: Could you also leave a comment, something like: 
# The golang-1.13 image used in OpenShift CI enforces vendoring and go get is disabled. Workaround that by unsetting it.

Comment (pawanpinjarkar) [reply hack/functest.sh:17]: Done

**Pull request merged**